src/task.js

```diff
@@ -0,0 +1,49 @@
+class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
```

Comment:
By default, new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length 🐐 Well done!